docs/pages/iconography_and_svgs.js

```diff
@@ -1,17 +1,18 @@
 // @flow strict
-import { useState, type Node } from 'react';
+import { useState, type Node, type ElementProps } from 'react';
 import { Box, ComboBox, Flex, Icon, TapArea, Layer, Toast, Tooltip } from 'gestalt';
 import MainSection from '../components/MainSection.js';
 import Page from '../components/Page.js';
 import PageHeader from '../components/PageHeader.js';
 
-function ClickableIcon({ iconName, onTap }: {| iconName: string, onTap: () => void |}) {
+type IconName = $NonMaybeType<$ElementType<ElementProps<typeof Icon>, 'icon'>>;
```

Comment:
Interesting. This is all basically what I was doing, with the exception of the `$NonMaybeType` and the stringification. Could you walk us through your debugging process for this in our next meeting?